### PR TITLE
Add GPG app

### PR DESCRIPTION
**Pull request by jarrodldavis**
Add an app entry for [probot-gpg](https://github.com/jarrodldavis/probot-gpg).

Closes jarrodldavis/probot-gpg#34

### CODE REVIEW

**Comment by jarrodldavis** [comment]
I need to add some meaningful documentation before this can be merged.

File: _apps/gpg.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: GPG
+description: Enforce GPG signatures on Pull Requests
+slug: probot-gpg
```
Comment (bkeepers) [inline _apps/gpg.md]: What would you think about just calling the app `gpg`? We've been trying to use really basic names (like `dco`, `welcome`, `stale`, etc) to set the precedent that these apps should aim to be feature complete.

**Comment by bkeepers** [comment]
One other thing that might be worth linking to: [Automatic Git commit signing with GPG on OSX](https://gist.github.com/bmhatfield/cc21ec0a3a2df963bffa3c1f884b676b)

File: _apps/gpg.md (hunk 1 above)
Comment (jarrodldavis) [reply _apps/gpg.md]: Okay I’ll rename my GitHub App and update this in a bit.

**Comment by bkeepers** [comment]
Thanks! We’ll tweet about it on Monday!

**Pull request merged**